packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -53,12 +55,20 @@ const ExperimentMainContent = ( {
 		return <Spinner />;
 	}
 
+	const description = navigationMenu
+		? sprintf(
+				/* translators: %s: The name of a menu. */
+				__( 'Structure for navigation menu: %s' ),
+				navigationMenu.title
```

Comment:
We should handle a situation where this `title` could be undefined. Just in case. I think I saw @talldan catch a similar bug on the Page List recently.